deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/deployment.yaml

```diff
@@ -26,10 +26,10 @@ spec:
               mountPath: /data
           resources:
             limits:
-              cpu: "6"
+              cpu: "7"
               memory: 60Gi
             requests:
-              cpu: "6"
+              cpu: "7"
```

Comment:
Please do not increase `helga`'s CPU until we confirm that this is a reasonable change. We may need to up workers to beefier CPU optimised types.